<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>808</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget
{
	border-image: url(:/new/img/图片/img/banner.jpg);
}
QFrame#frame
{
border:0px groove gray;border-radius:40px;padding:2px 4px;
	border-image: url(:/new/img/图片/img/music-back.png);
};</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>831</width>
      <height>481</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="getSong">
     <property name="geometry">
      <rect>
       <x>715</x>
       <y>215</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color:transparent;
border-image: url(:/new/icon/图片/icon/音乐列表-01 (1).png);
border:0px groove gray;border-radius:15px;padding:2px 4px;
}


QPushButton#getSong:hover{  
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(221, 24, 255, 255), stop:1 rgba(255, 17, 48, 255));

border:0px groove gray;border-radius:15px;padding:2px 4px;    

} 
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="next_Btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>135</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>465132</width>
       <height>645132</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/icon/图片/icon/icons8-fast-forward-48.png);
border:0px groove gray;border-radius:15px;padding:2px 4px;
}
QPushButton#next_Btn:hover{  
	border-image: url(:/new/icon/图片/icon/icons8-fast-forward-48.png);
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 243, 255, 255), stop:1 rgba(173, 165, 255, 255));
border:0px groove gray;border-radius:15px;padding:2px 4px;    

} ;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="play_Btn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>135</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/icon/图片/icon/icons8-resume-button-48.png);
border:0px groove gray;border-radius:15px;padding:2px 4px;
}
QPushButton#play_Btn:hover{  
	border-image: url(:/new/icon/图片/icon/icons8-resume-button-48.png);
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 243, 255, 255), stop:1 rgba(173, 165, 255, 255));

border:0px groove gray;border-radius:15px;padding:2px 4px;    

} 
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pre_Btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>135</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/icon/图片/icon/icons8-rewind-48.png);
border:0px groove gray;border-radius:15px;padding:2px 4px;
}
QPushButton#pre_Btn:hover{  
border-image: url(:/new/icon/图片/icon/icons8-rewind-48.png);
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 243, 255, 255), stop:1 rgba(173, 165, 255, 255));
border:0px groove gray;border-radius:15px;padding:2px 4px;    

} </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>211</width>
       <height>201</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>171</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/new/img/图片/img/1.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>230</y>
       <width>43</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>00：00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_voice">
     <property name="geometry">
      <rect>
       <x>655</x>
       <y>140</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/icon/图片/icon/icons8-voice-48.png);
border-radius:15px;
}
QPushButton#Btn_voice:hover{
border-image: url(:/new/icon/图片/icon/icons8-voice-48.png);
border-radius:15px;
	
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(231,80,229, 255), stop:1 rgba(7,208,255, 255))!important;

};</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>655</x>
       <y>220</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/new/icon/图片/icon/顺序循环.png);
	border:0px groove gray;border-radius:15px;padding:2px 4px;
}

QPushButton#pushButton_2:hover{  
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(19, 232, 255, 255), stop:1 rgba(158, 255, 46, 255));

border:0px groove gray;border-radius:15px;padding:2px 4px;    

} 
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Local_song">
     <property name="geometry">
      <rect>
       <x>645</x>
       <y>70</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:transparent;
border-image: url(:/new/icon/图片/icon/文件.png);
border:0px groove gray;border-radius:15px;padding:2px 4px;
}


QPushButton#Local_song:hover{  

  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(69, 255, 162, 255), stop:1 rgba(244, 255, 35, 255));

border:0px groove gray;border-radius:15px;padding:2px 4px;    

} 
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_song">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 12pt &quot;Sans Serif&quot;;
color: rgb(170, 255, 255);

</string>
     </property>
     <property name="text">
      <string>歌曲名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_singer">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 11pt &quot;Sans Serif&quot;;
color: rgb(170, 255, 255);</string>
     </property>
     <property name="text">
      <string>歌手名字</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPos">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px solid rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>00：00</string>
     </property>
    </widget>
    <widget class="QSlider" name="volume">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>135</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"> QSlider::add-page:Horizontal
                                      {     
                                         background-color: rgb(87, 97, 106);
                                         height:4px;
                                      }
QSlider::sub-page:Horizontal 
                                     {

                                         background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 255, 255, 255), stop:1 rgba(7,208,255, 255));
                                         height:4px;
                                      }
QSlider::groove:Horizontal 
                                     {
                                         background:transparent;
                                         height:6px;
                                     }
 QSlider::handle:Horizontal 
                                     {
                                         height: 15px;
                                         width:15px;
										                                   
	
	border-image: url(:/new/icon/图片/icon/耳机.png);

                                         margin: -8 0px; 
                                     }</string>
     </property>
     <property name="maximum">
      <number>101</number>
     </property>
     <property name="pageStep">
      <number>11</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>771</width>
       <height>3</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">/*border-color: rgb(215, 252, 255);
color: rgb(182, 164, 255);
color: rgb(85, 255, 255);
color: rgb(0, 0, 255);
background-color: rgb(190, 255, 248);*/
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 255, 255, 255), stop:1 rgba(182, 164, 255, 255));</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="musicPlan">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>190</y>
       <width>541</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"> QSlider::add-page:Horizontal
                                      {     
                                         background-color: rgb(87, 97, 106);
                                         height:4px;
                                      }
QSlider::sub-page:Horizontal 
                                     {

                                         background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 255, 255, 255), stop:1 rgba(7,208,255, 255));
                                         height:4px;
                                      }
QSlider::groove:Horizontal 
                                     {
                                         background:transparent;
                                         height:6px;
                                     }
 QSlider::handle:Horizontal 
                                     {
                                         height: 15px;
                                         width:15px;
	border-image: url(:/new/icon/图片/icon/雪花.png);
                                         margin: -8 0px; 
                                     }</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="ShowToUser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>771</width>
       <height>181</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget#ShowToUser
{
border-radius:20px;
font: 14pt &quot;Sans Serif&quot;;
color: rgb(170, 255, 255);
background-color: rgba(33, 33, 33,100);
}
QScrollBar:vertical
{
    width:8px;
    
	background-color: rgb(0, 255, 127);
    margin:0px,0px,0px,0px;
    padding-top:9px;   // 留出9px给上面和下面的箭头
    padding-bottom:9px;
	border-image: url(:/new/img/图片/img/preview.jpg);
}
QScrollBar::handle:vertical
{
    width:8px;
    background:rgba(0,0,0,25%);
    border-radius:4px;   // 滚动条两端变成椭圆
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(0,0,0,50%);   // 鼠标放到滚动条上的时候，颜色变深
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical   // 这个应该是设置下箭头的，3.png就是箭头
{
    height:9px;width:8px;
    border-image:url(:/images/a/3.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical   // 设置上箭头
{
    height:9px;width:8px;
    border-image:url(:/images/a/1.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover   // 当鼠标放到下箭头上的时候
{
    height:9px;width:8px;
    border-image:url(:/images/a/4.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover  // 当鼠标放到下箭头上的时候
{
    height:9px;width:8px;
    border-image:url(:/images/a/2.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical   // 当滚动条滚动的时候，上面的部分和下面的部分
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLen">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>  00：00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Find">
     <property name="geometry">
      <rect>
       <x>645</x>
       <y>20</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:transparent;
border-image: url(:/new/icon/图片/icon/搜索.png);
border:0px groove gray;border-radius:15px;padding:2px 4px;
}


QPushButton#btn_Find:hover{  

  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(221, 24, 255, 255), stop:1 rgba(255, 17, 48, 255));

border:0px groove gray;border-radius:15px;padding:2px 4px;    

} 
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Sink">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>70</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:transparent;
border-image: url(:/new/icon/图片/icon/皮肤.png);
border:0px groove gray;border-radius:15px;padding:2px 4px;
}


QPushButton#btn_Sink:hover{  

  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(221, 24, 255, 255), stop:1 rgba(255, 17, 48, 255));

border:0px groove gray;border-radius:15px;padding:2px 4px;    

} 
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="CMB_song">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_FIND">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Btnclose">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>45</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/new/icon/图片/icon/关闭.png);
border-radius:15px;
}
QPushButton#Btn_voice:hover{
border-image: url(:..new/icon/图片/icon/关闭.png);
border-radius:15px;
	
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(231,80,229, 255), stop:1 rgba(7,208,255, 255))!important;

};</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
